output/baseoutput/clientsession.go

```diff
@@ -120,6 +123,9 @@ func (session *clientSession) resendLeftovers(leftovers chan base.LogChunk) (cha
 func (session *clientSession) processInput(maxDuration time.Duration) (chan base.LogChunk, reconnectPolicy) {
 	maxSessionDurationSignal := time.After(maxDuration)
 
+	reconnectChan := make(chan os.Signal, 10)
+	signal.Notify(reconnectChan, syscall.SIGUSR1)
```

Comment:
so SIGUSR1 and SIGUSR2 can be used for anything according to https://www.gnu.org/software/libc/manual/html_node/Miscellaneous-Signals.html#:~:text=The%20SIGUSR1%20and%20SIGUSR2%20signals,SIGUSR2%20in%20Signaling%20Another%20Process.

Are there any concerns about another program or process raising this signal?

Reply:
Hopefully even if this signal is used somewhere else (in other RELEX applications) - it's not broadcasted to all the processes on the machine, so we can expect not to catch random SIGUSRs in production.

Reply:
What happens when there are multiple sessions? Does sending one signal trigger all of them?